CHANGELOG.md

```diff
@@ -19,7 +19,9 @@
 * New grayscale style. This also add two new cmaps `cet_grey_r` and `cet_grey_r`. These are perceptually uniform gray scale cmaps from colorcet (linear_grey_10_95_c0) (#1164)
 * Add warmup groups to InferenceData objects, initial support for PyStan (#1126) and PyMC3 (#1171)
 
+* Integrate jointplot into pairplot, add point-estimate and overlay of plot kinds #1079
```

Comment:
this line has been triplicated, it is also 3 points above and the 3rd point starting from the top